plinth/templates/setup.html

```diff
@@ -50,8 +50,17 @@
       <form action="" method="post">
         {% csrf_token %}
 
-        <input type="submit" class="btn btn-md btn-primary"
-               value="{% trans "Install" %}" />
+        {% if dpkg_is_active %}
+          <div class="alert alert-warning" role="alert">
+            Another installation or upgrade is already running.
+            Please wait for a few moments and refresh this page.
```

Comment:
Thanks, done; could have spotted that myself.
I've also slightly rewritten the 2nd sentence: "Please wait for a few moments before trying again." in case users aren't aware it's a "(web) page" they're looking at, and to slightly encourage that they can browse elsewhere in the meantime.